src/Commands/ArmPos.cpp

```diff
@@ -0,0 +1,53 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "ArmPos.h"
+
+ArmPos::ArmPos(int position) : mPosition(position), counter(0) {
+	// Use Requires() here to declare subsystem dependencies
+	// eg. Requires(Robot::chassis.get());
+	Requires(Robot::arm.get());
+}
+
+// Called just before this Command runs the first time
+void ArmPos::Initialize() {
+
+}
+
+// Called repeatedly when this Command is scheduled to run
+void ArmPos::Execute() {
+	Arm::getInstance()->SetArmPos(mPosition);
+}
+
+// Make this return true when this Command no longer needs to run execute()
+bool ArmPos::IsFinished() {
+	if (abs(Arm::getInstance()->GetPosError()) < 1000
+			|| Arm::getInstance()->IsFwdLimitSwitchClosed()
+			|| Arm::getInstance()->IsRevLimitSwitchClosed()) {
+		if (counter > 29) {
```

Comment:
Le't's be careful with tabs and spaces.

Reply:
What is `29`? 

I think this is determined later, so let's just leave this. But we need to name the value and say how we determined it's value.

Reply:
`29` is how many times does the arm have to be in the right error before the arm is in the right position. 

We need a name for that. 😄